parse/apt_test.go

```diff
@@ -11,10 +11,15 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-package parse
+package parse_test
```

Comment:
Any reason to change the package? That seems odd.

Reply:
I stumbled across a couple "go testing best practices" articles that suggested leveraging this golang feature that allows test .go files to be in a "_test" package in the same folder. Thought I'd give it a "test" - all puns intended. :)

https://hackernoon.com/5-advanced-go-testing-techniques-7199b686b6c1
snippet:

Use a separate _test package

Most tests in the ecosystem are created in files pkg_test.go but still live in the same package: package pkg. A separate test package is a package you create in a new file, foo_test.go, inside the directory of the package you wish to test, foo/, with the declaration package foo_test . From there, you can import github.com/example/foo and other dependencies. This feature enables a number of things. It is the recommended workaround for cyclic dependencies in tests, it prevents brittle tests, and it allows the developer to feel what it’s like to consume their own package. If your package is hard to use, it will likely also be hard to test using this method.